packages/ignite/src/plugins/default/commands/add.js

```diff
@@ -1,8 +1,7 @@
 // @cliDescription  Add a new thingy
 // ----------------------------------------------------------------------------
-// const Exists = require('npm-exists')
+
 const Toml = require('toml')
-// Yeah, why would toml include this? :(
```

Comment:
you killed my passive agressive toml angst.   Now I have to dye my hair black and join a goth band.